### PR TITLE
Readme and yaml updates

### PR DESCRIPTION
**Pull request by alexiswl**
Conda env:  
* Added aws2-wrap to conda env   

README:
* Updated README to include staging data segment.  
* Removed sbcast notes.  
* Removed unset mount points notes.  
* Added segment on uploading data back to s3.

### CODE REVIEW

File: parallel_cluster/README.md [hunk 1]
```diff
@@ -93,6 +93,20 @@ from the `ec2-user`, this is handy for debugging purposes:
...
+Ensure that inputs are accessible to all nodes by placing it in the `${SHARED_DIR}` folder.  
+This is:
+  * `/efs` if using the efs configuration on `tothill` or `umccr_dev`
+  * `/fsx` if using the fsx configuration on `umccr_dev_fsx`
+  
+By default you will have read-only access to the s3 buckets.  
+You may use `aws s3 sync s3://<bucket_path> "${SHARED_DIR}/local_path` to download data
+into the shared file system.  
+
+> Compute nodes have a much higher bandwidth than the master nodes.
```
Comment (brainstorm) [inline parallel_cluster/README.md:107]: I would put that bigger/bold for users to notice... along with an sbatch script example for "data copy purposes".

File: parallel_cluster/README.md [hunk 2]
```diff
@@ -208,6 +210,30 @@ The following worklows are working examples you can run through to see AWS Paral
...
 [bcbio variant calling example](working_examples/bcbio/bcbio-nextgen.md)
 
+## Uploading data back to s3
+
+By default, parallel cluster does not have write access to s3 buckets.  
+A workaround is taking your short-term local SSO credentials and importing them into parallel cluster.  
+
+From your local computer run:
+```
+aws2-wrap --profile "${AWS_PROFILE}" --export | xclip  # pbcopy if on a mac.
```
Comment (brainstorm) [inline parallel_cluster/README.md:220]: Not 100% sure if we want to introduce yet another tool to do this when:

1) I showed @AndrewPattison how to do it via the UMCCR AWS SSO apps portal (https://umccr.awsapps.com/start) and he was fine with it?
2) We already have [`yawsso`](https://github.com/victorskl/yawsso/) from @victorskl to attain precisely that too? :-S

**Review by brainstorm — CHANGES_REQUESTED** [review]

File: parallel_cluster/README.md (hunk 2 above)
Comment (AndrewPattison) [reply parallel_cluster/README.md:220]: I was ok with it but not leaving the command line would be ideal

Comment (victorskl) [reply parallel_cluster/README.md:220]: oh wow! yet another aws sso! 😅   -- yap right yawsso export creds token too!

Comment (alexiswl) [reply parallel_cluster/README.md:220]: The pcluster conda env uses aws 1.18 though, yawsso relies on aws v2 being installed. From a recent AWS thread, I doubt aws v2 will come to pip any time soon. Can we stick with `aws2-wrap` for now? The portal approach is difficult to describe in a readme without a series of screen shots.

```
(pcluster) $ yawsso -p "${AWS_PROFILE}"
Required AWS CLI v2. Found aws-cli/1.18.121 Python/3.8.5 Linux/4.19.84-microsoft-standard botocore/1.17.44
(pcluster) $ aws2-wrap --profile "${AWS_PROFILE}" --export
export AWS_ACCESS_KEY_ID="CD"
export AWS_SECRET_ACCESS_KEY="AB"
export AWS_SESSION_TOKEN="XTZ"
export AWS_REGION="ap-southeast-2"
```

Comment (brainstorm) [reply parallel_cluster/README.md]: Would it make sense to uninstall that deprecated aws cli from pcluster and force-install AWS v2 through image builder? That would make more sense (to me) that introducing yet another AWS SSO tool that has already been "yet-anotherized" (by Victor) :-S

Comment (alexiswl) [reply parallel_cluster/README.md]: This is the local conda env that runs the `pcluster create`, `pcluster list` commands, not the AMI on which the cluster is built on (which is aws2 btw).

Comment (alexiswl) [reply parallel_cluster/README.md]: Simply put, until [this issue](https://github.com/aws/aws-cli/issues/4947) goes away, we're bound on aws v1 on this conda env. It does make it annoying to run `aws sso` whilst on that environment, so I have aliases set up for that.

Comment (alexiswl) [reply parallel_cluster/README.md]: In my global env, The command `aws2` is soft-linked to `aws`. @victorskl could `yawsso` could first search for `aws2` and then fall back to `aws`? Similar to `python3` to `python` soft-links

Comment (alexiswl) [reply parallel_cluster/README.md]: Nm, I can see that can be done with the -b flag

Comment (alexiswl) [reply parallel_cluster/README.md]: `yawsso --profiles "${AWS_PROFILE}" --bin "$(which aws2)" --export-vars` is an alternative but requires users to have the `aws2 -> aws` softlink set up.

**Comment by brainstorm** [comment]
We moved the parallel_cluster folder over to https://github.com/umccr/aws_parallel_cluster... I should have merged this one first :-S

**Pull request closed without merging**